scripts/bundles/cli.ts

```diff
@@ -45,6 +45,28 @@ export async function cli(opts: BuildOptions): Promise<ReadonlyArray<RollupOptio
   dts = dts.replace('@stencil/core/internal', '../internal/index');
   await fs.writeFile(join(opts.output.cliDir, dtsFilename), dts);
 
+  // bundle config-flags.js
+  const esConfigFlagsOutput: OutputOptions = {
+    format: 'es',
+    file: join(outputDir, 'config-flags.js'),
+    preferConst: true,
+    sourcemap: true,
+    banner: getBanner(opts, `Stencil CLI Config Flags`, true),
+  };
+
+  const cjsConfigFlagsOutput: OutputOptions = {
+    format: 'cjs',
+    file: join(outputDir, 'config-flags.cjs'),
+    preferConst: true,
+    sourcemap: true,
+    banner: getBanner(opts, `Stencil CLI Config Flags (CommonJS)`, true),
+  };
```

Comment:
I don't think something's right here - if I build this branch, I see the generated `config-flags` files, but it looks like they define/re-export functions like `run`, `runTask`.  Those functions are _also_ defined in `index.js/cjs`. Is that intentional?

Reply:
hmm that's not what I intended...

what I _wanted_ to do was just add this new `config-flags` module but not have to add a new file to the build, by just having it get bundled into the `index.js` output for the whole `cli` folder but doing an `export * from ...` pointing to the `config-flags` module lead to a bunch of other errors...I'll play around with it a little more and see what I can figure out.